api/src/main/java/mcoffin/rogue/api/MecanumDrive.java

```diff
@@ -0,0 +1,5 @@
+package mcoffin.rogue.api;
+
+public interface MecanumDrive {
+  public void vectorMove(double forwardSpeed, double horizontalSpeed, double turnSpeed);
```

Comment:
Since we now have `DriveVector`, `vectorMove` should take one as input instead of two doubles.